vpn-server.yml

```diff
@@ -0,0 +1,4 @@
+---
+- hosts: dc1:&vpn_servers
```

Comment:
Ok, I'll add the same group into dc2 inventory. User will be able to choose necessary servers with the inventory parameter.